lib/static/components/section/body/meta-info/content.jsx

```diff
@@ -95,8 +95,19 @@ class MetaInfoContent extends Component {
             formattedMetaInfo[key] = {content: formattedMetaInfo[key]};
         });
 
-        if (result.suiteUrl) {
-            formattedMetaInfo.url = {content: result.metaInfo.url || result.suiteUrl, url: getUrlWithBase(result.suiteUrl, baseHost)};
+        for (const [key, value] of Object.entries(formattedMetaInfo)) {
```

Comment:
Hm, I'm not sure that we should replace base host for all urls in meta. I believe that by default it should only be used for `url` field and other fields should be specified by user through `metaInfoBaseUrls` option.